src/pages/Login.js

```diff
@@ -0,0 +1,86 @@
+import React, { useEffect, useState } from "react";
+import { useDispatch, useSelector } from "react-redux";
+import { Link, useNavigate } from "react-router-dom";
+import { login } from "../features/authSlice";
+import { useLocation } from "react-router-dom";
+
+export const Login = () => {
+  const [form, setForm] = useState({ username: "", password: "" });
```

Comment:
Can rename this state variable as [loginDetails,setLoginDetails] for better understanding later on